Testscripts/Linux/nginx-community-tests.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+###############################################################################
+#
+# Description:
+# For the nginx test,the packages installed by yum
+#   cpan hg pcre geoip perl-Geo-IP.x86_64
+#   GeoIP-devel.x86_64 memcached perl-FCGI.x86_64
+#   perl-GDGraph.noarch perl-ExtUtils-Embed.noarch
+#
+# For the nginx test,the perl packages installed
+#   Test::Nginx Protocol::WebSocket
+#   IO::Socket::SSL Cache::Memcached
+#   Cache::Memcached::Fast
+#
+###############################################################################
+
+# Source utils.sh
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+# Checks what Linux distro we are running on
+GetDistro
+update_repos
+
+LogMsg "Installing dependencies"
+common_packages=(m4 bison flex make gcc psmisc autoconf automake)
+
+update_repos
+install_package "${common_packages[@]}"
+
+case $DISTRO in
+    "ubuntu"*)
+        deb_packages=(git libaio-dev libattr1 libcap-dev keyutils \
+            libdb4.8 libberkeleydb-perl expect dh-autoreconf gdb \
+            libnuma-dev quota genisoimage db-util unzip exfat-utils)
+        install_package "${deb_packages[@]}"
+        ;;
+    *)
+        LogMsg "Unknown distro $DISTRO, continuing to try for RPM installation"
+        ;;
+esac
+
+Timeout()
+{
+    waitfor=3600
+    touch nginx-test.log
+    TEST_NGINX_BINARY=$execute_path/nginx prove . >$SCRIPTPATH/nginx-test.log &
+    commandpid=$!
+
+    ( sleep $waitfor ; kill -9 $commandpid  > /dev/null 2>&1 ) &
+    sleeppid=$PPID
+
+    wait $commandpid > /dev/null 2>&1
+    kill $sleeppid > /dev/null 2>&1
+    return 0
+}
+
+#pre-set the env
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+nginx_git_url="https://github.com/nginx/nginx.git"
```

Comment:
Those 2 git repositories do not have the license. We should not add into LISAv2. 1) you will find their license file, or 2) find an alternative.